tapir/core/tapir_email_base.py

```diff
@@ -17,9 +17,17 @@
 
 all_emails: Dict[str, Type[TapirEmailBase]] = {}
 
+enabled_by_default_options = Literal[True, False, "both"]
 
-def get_all_emails(default: bool | None = True) -> list[Type[TapirEmailBase]]:
-    if default is not None:
+
+def get_all_emails(
+    default: enabled_by_default_options = True,
+) -> list[Type[TapirEmailBase]]:
+    """
+    default="both" returns both, default and non-default mails.
+    default=False returns mails not being sent by default
+    """
+    if default != "both":
```

Comment:
I think it would be clearer like this:
```python
if default == "both:
    return [mail for mail in TapirEmailBase.__subclasses__()]

return [mail for mail in TapirEmailBase.__subclasses__() if mail.enabled_by_default is default]
```